src/_base/harness/scripts/xdebug_version_sync.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+PREPARE_CONFIG=0
```

Comment:
```suggestion
set -e -o pipefail

PREPARE_CONFIG=0
```
To stop execution if any command fails.